cardano-cli/src/Cardano/CLI/Types/Errors/AddressCmdError.hs

```diff
@@ -0,0 +1,44 @@
+{-# LANGUAGE CPP #-}
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+
+module Cardano.CLI.Types.Errors.AddressCmdError
```

Comment:
I am not sure about creating separate `Errors` module. I think the error type should lie within its domain and near the code using it. 

Whats the benefit here?